language-support/ts/codegen/tests/build-and-lint.sh

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2019 The DAML Authors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+
+YARN=$PWD/$1
```

Comment:
You probably need to add some bazel runfiles magic to make this work on Windows, the tests for `daml trigger --list` might be helpful as an example https://github.com/digital-asset/daml/blob/master/triggers/tests/list-triggers.sh